### ISSUE TITLE
Marker with Label only last marker is displayed -- AGAIN

### ISSUE THREAD

**Issue by GoogleCodeExporter** [seancallinan/google-maps-utility-library-v3#320]
```
http://google-maps-utility-library-v3.googlecode.com/svn/tags/markerwithlabel/1.
1.8/examples/basic.html

The above marker with label example link only shows the last marker with both 
labels showing -- AGAIN. This happened last March and is happening again. I was 
using &v=3.15 to correct the problem but now even that does not work.


```

Original issue reported on code.google.com by `mike.sae...@gmail.com` on 27 Aug 2014 at 5:39

**Comment by GoogleCodeExporter** [comment]
```
I believe this is because the MarkerWithLabel object is inheriting from 
google.maps.Marker. I had code that did this and it was broken after 3.15. I 
refactored the code not to directly inherit and it started working again.

However, this isn't my library.
```

Original comment by `chadkillingsworth` on 27 Aug 2014 at 2:01

**Comment by GoogleCodeExporter** [comment]
```
This problem is fixed since version 1.1.9:
Example:
http://google-maps-utility-library-v3.googlecode.com/svn/tags/markerwithlabel/1.
1.9/examples/basic.html
```

Original comment by `jcarlosdiaz` on 1 Sep 2014 at 7:28
